src/os/bluestore/BlueStore.cc

```diff
@@ -1451,26 +1451,91 @@ void BlueStore::Blob::discard_unallocated(Collection *coll)
 }
 
 void BlueStore::Blob::get_ref(
-  uint64_t offset,
-  uint64_t length)
+  Collection *coll,
+  uint32_t offset,
+  uint32_t length)
 {
-  ref_map.get(offset, length);
+
+  used_in_blob.get(
+    offset,
+    length,
+    [&](uint32_t* cnt, uint32_t* rsz) {
+      uint32_t min_release_size = blob.get_release_size(coll->store->min_alloc_size);
+    // fallback might be forbidden due to compression, min_release_size = 0 in this case
+      if (min_release_size) {
+        uint64_t l = blob.get_logical_length();
+	*cnt = ROUND_UP_TO(l, min_release_size) / min_release_size;
+	*rsz = min_release_size;
+      } else {
+	*cnt = *rsz = 0;
```

Comment:
since both callers initialize these to 0 we could drop these blocks and make it part of the calling convention.